ti/instrumentation/java/bazel.go

```diff
@@ -100,6 +174,8 @@ func (b *bazelRunner) GetCmd(ctx context.Context, tests []ti.RunnableTest, userA
 		return "echo \"Skipping test run, received no tests to execute\"", nil //nolint:goconst
 	}
 
+	tests = getBazelTestRules(ctx, b.log, tests)
```

Comment:
Do we still need to do stuff in lines 184-255? I thought we are already getting all the test rules here.